CHANGELOG.md

```diff
@@ -18,6 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - tests: new Task targets `test:unit` (replaces target `test`) and `test:integration` (replaces target `test-integration`).
 - build: cleaner Taskfile.
 
+### Fixed
+
+- return error text code in case of http code 500. 
```

Comment:
The other entries start with an Uppercase letter.
```suggestion
- Return error text code in case of http code 500. 
```

Reply:
Thanks, corrected.